saleor/graphql/order/mutations/orders.py

```diff
@@ -670,6 +671,7 @@ def perform_mutation(cls, _root, info, **data):
         cls.validate_variants(order, variants)
 
         lines = cls.add_lines_to_order(order, lines_to_add)
+        allocate_lines(lines)
```

Comment:
Correct me if I am wrong but you unify the logic for unconfirmed and draft orders. Am I right? I don't think that it is expected behaviour to allocate draft order in the same way as the unconfirmed order.